cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -1430,22 +1430,22 @@ func TestFilterOutYoungPods(t *testing.T) {
 			newPodScaleUpDelay: 5 * time.Minute,
 			runTime:            now,
 			pods:               []*apiv1.Pod{p1, p2},
-			expectedPods:       []*apiv1.Pod{},
+			expectedPods:       []*apiv1.Pod(nil),
 		},
 		{
 			name:               "annotation delay smaller than global",
 			newPodScaleUpDelay: 5 * time.Minute,
 			runTime:            now,
 			pods:               []*apiv1.Pod{p1, p3},
```

Comment:
`p3` has annotation of 2m and is created 1m before `runTime`, so it wouldn't be included even if the implementation allowed for annotation-based delay to be smaller than the global one. I think this test should use `runTime: now.Add(2 * time.Minute)` to actually show the difference.

Reply:
Yes, that is actually very good point. Thank you!